fizzbuzz/fizzbuzz.c

```diff
@@ -10,7 +10,27 @@
 */
 int fizzbuzz(int n)
 {
-
+    int count = 0;
+    for (int i = 0; i < n; i++)
+    {
+        if (i % 3 == 0)
+        {
+            printf("Fizz\n");
+        }
+        else if (i % 5 == 0)
+        {
+            printf("Buzz\n");
+        }
+        else if (i % 5 == 0 && i % 3 == 0)
```

Comment:
You will probably want to make this the first conditional in your for loop. Otherwise it will never be reached because `if (i % 3 == 0)` will be triggered and the looped will be exited before it can get to `else if (i % 5 == 0 && i % 3 == 0)` for numbers divisible by both 3 and 5